vnext/overrides.json

```diff
@@ -28,6 +28,12 @@
       "baseHash": "2aed54df1dcb93f3e4366eb083804af541aa8b13",
       "issue": 7591
     },
+    {
+      "type": "derived",
```

Comment:
@NickGerleman To make sure this is the right overrides category

Reply:
Also, do you know the flow check is failing here?
It says the new `isPressable` prop is missing in NativeVirtualText - seems like the forked file is not getting picked up by Flow. I've tried adding the base file to the ignore enumeration in `.flowconfig` but it didn't help

Reply:
Fixed by adding to .flowconfig. @NickGerleman please let me know if I should use `patch` instead of `derived` in the overrides.json file.